.github/workflows/core-hadoop2-ci.yml

```diff
@@ -38,10 +38,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Set up JDK 8
+      - name: Set up JDK 11
```

Comment:
Do we still need the CI of JDK8?

After the change, it seems that we can't guarantee that Amoro can still work on JDK8.

IIUC, we need to support running CI on both JDK8 and JDK11 when we start deprecating JDK8, and the CI of JDK8 can be removed when we no longer support JDK8.